src/main/scala/com/github/unchama/seichiassist/subsystems/breakflags/domain/BreakFlagName.scala

```diff
@@ -0,0 +1,24 @@
+package com.github.unchama.seichiassist.subsystems.breakflags.domain
+
+import enumeratum.{Enum, EnumEntry}
+
+/**
+ * 定義されたブロックを破壊するかしないかを表すためのフラグの名前を集めたEnum
+ */
+sealed trait BreakFlagName extends EnumEntry
+
+object BreakFlagName extends Enum[BreakFlagName] {
+
+  val values: IndexedSeq[BreakFlagName] = findValues
+
+  /**
+   * Chestブロックを破壊するかを示すフラグ名
+   */
+  case object Chest extends BreakFlagName
+
+  /**
+   * ネザークォーツ系ブロックを破壊するかを示すフラグ名
+   */
+  case object NetherQuartz extends BreakFlagName
```

Comment:
```suggestion
  case object MadeFromNetherQuartz extends BreakFlagName
```
上記のことを命名にも反映したほうが良さそう